src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/ruleNode.tsx

```diff
@@ -278,13 +297,41 @@ class RuleNode extends React.Component<Props> {
   }
 
   render() {
-    const {data, disabled} = this.props;
+    const {data, disabled, node} = this.props;
+    const ticketRule = node?.hasOwnProperty('actionType');
 
     return (
       <RuleRowContainer>
         <RuleRow>
-          {data && <input type="hidden" name="id" value={data.id} />}
-          {this.renderRow()}
+          <Rule>
+            {data && <input type="hidden" name="id" value={data.id} />}
+            {this.renderRow()}
+            {ticketRule && (
+              <Button
+                size="small"
+                icon={<IconSettings size="xs" />}
+                onClick={() => this.openModal()}
+              >
+                {
+                  <Modal
```

Comment:
Can you move the modal to a new file?

Reply:
@mgaeta I can't tag you to review this since you created the initial commit but I separated it, can you check to make sure it's okay?